assets/dev/js/editor/controls/code.js

```diff
@@ -101,6 +103,10 @@ ControlCodeEditorItemView = ControlBaseDataView.extend( {
 	onDestroy() {
 		elementor.panel.$el.off( 'resize.aceEditor' );
 	},
+
+	isEditable() {
+		return ! elementor.userCan( 'custom-html' );
```

Comment:
maybe I should check if it's actually HTML widget

Reply:
weird thing - `elementor.userCan( 'custom-html' )` is returning reverted state of 'custom-html' checkbox in role manager. As example when it's unchecked - custom html shouldn't be available, userCan returns `true`